docs: rewrite SAML SSO guide and split into dedicated sub-pages (#7484)
elastic/docs-content: deploy-manage/users-roles/cluster-or-deployment-auth/saml-entra.md

        * For `idp.metadata.path`, we’ve shown the format to construct the URL. This value should be identical to the `App Federation Metadata URL` setting that you made a note of in the previous step.

        :::{admonition} For organizations with many group memberships

>         If you configure [`attributes.groups`](/deploy-manage/users-roles/cluster-or-deployment-auth/saml-attribute-mapping.md#saml-es-user-properties) to read the list of Azure AD groups from the SAML assertion, be aware that users who belong to many groups may exceed Azure AD’s size limit for SAML tokens. In that case, the `groups` attribute will be omitted.

Reviewer: little style edit while we're here
```suggestion
        If you configure [`attributes.groups`](/deploy-manage/users-roles/cluster-or-deployment-auth/saml-attribute-mapping.md#saml-es-user-properties) to read the list of Azure AD groups from the SAML assertion, users who belong to many groups might exceed Azure AD’s size limit for SAML tokens. In that case, the `groups` attribute is omitted.
```
